dotcom-rendering/src/components/EuropeLandingModal.importable.tsx

```diff
@@ -196,6 +197,7 @@ export const EuropeLandingModal = ({ edition }: Props) => {
 		) {
 			localStorage.setItem(modalShownKey, 'true');
 			europeModal.showModal();
+			europeModal.click();
```

Comment:
This is here to send the Ophan click event when the modal opens @SHession

Reply:
Discussed with @AshCorr, should be fine as a temporary solution.